### ISSUE TITLE
gist/standalone-example fails to build: error: no matching function for call to 'branch'

### ISSUE THREAD

**Issue by yurivict** [Gecode/gecode#103]
**Describe the bug**

clang-10 fails to compile it:
```
mygist.cpp:47:5: error: no matching function for call to 'branch'
    branch(*this, x, INT_VAR_NONE, INT_VAL_MIN);
    ^~~~~~
../../../gecode/int.hh:5071:3: note: candidate function not viable: no known conversion from 'Gecode::IntVarBranch ()' to 'Gecode::IntVarBranch' for 3rd argument
  branch(Home home, const IntVarArgs& x,
  ^
../../../gecode/int.hh:5081:3: note: candidate function not viable: no known conversion from 'Gecode::IntVarBranch ()' to 'TieBreak<Gecode::IntVarBranch>' for 3rd argument
  branch(Home home, const IntVarArgs& x,
  ^
```

gecode-6.2.0
clang-10
FreeBSD 12.2

**Comment by zayenz** [comment]
Thanks for reporting, the file seems to have been neglected for a long time. Will add a fix later.

If you want to fix it locally, the line should read
```
branch(*this, x, INT_VAR_NONE(), INT_VAL_MIN());
```

**Comment by zayenz** [comment]
Fixed in release/6.3.0 (85e2d67).
